services/accounts/accounts.go

```diff
@@ -15,21 +15,23 @@ import (
 	"github.com/status-im/status-go/multiaccounts/accounts"
 	"github.com/status-im/status-go/multiaccounts/settings"
 	"github.com/status-im/status-go/params"
+	"github.com/status-im/status-go/protocol"
 )
 
 const pathWalletRoot = "m/44'/60'/0'/0"
 const pathDefaultWallet = pathWalletRoot + "/0"
 
-func NewAccountsAPI(manager *account.GethManager, config *params.NodeConfig, db *accounts.Database, feed *event.Feed) *API {
-	return &API{manager, config, db, feed}
+func NewAccountsAPI(manager *account.GethManager, config *params.NodeConfig, db *accounts.Database, feed *event.Feed, messenger **protocol.Messenger) *API {
```

Comment:
I'm curious why are you passing in a pointer to pointer `messenger **protocol.Messenger`?